docker/gemini/Dockerfile

```diff
@@ -7,11 +7,11 @@ RUN apk add curl
 ARG version
 ENV version=${version:-1.7.6}
 
-RUN curl -LO https://s3.amazonaws.com/downloads.scylladb.com/gemini/${version}/gemini_${version}_Linux_x86_64.tar.gz ;\
+RUN curl -LO https://github.com/scylladb/gemini/releases/download/v${version}/gemini_${version}_Linux_x86_64.tar.gz ;\
     tar -xvf gemini_${version}_Linux_x86_64.tar.gz ;\
     chmod a+x gemini
 
-FROM bitnami/minideb:bullseye
+FROM ubuntu:22.04
```

Comment:
needed to match the glibc that is using in building gemini in github actions

Reply:
isn't minideb using glibc?

Reply:
btw. which github actions are you referring to?

Reply:
> isn't minideb using glibc?

It is but it wasn't the correct version

Reply:
> btw. which github actions are you referring to?

I thought it was built by CI

But looking at callstack from it, it was built on someone's local machine.

we should have a proper CI to build it